index.html

```diff
@@ -55,7 +55,7 @@
             >
           </li>
         </ul>
-        <a href="javascript:void(0);" class="icon" onclick="toggleMobileNav()">
+        <a id="hamburger" href="javascript:void(0);" class="icon" onclick="toggleMobileNav()">
```

Comment:
You've only added this on the index page, but not on the contact or activities page.
Please do so, too. This maintain consistency across all pages.
![image](https://github.com/rbjks/rbjks.github.io/assets/43912470/62683f92-313a-49b6-a9ae-1c50f723b051)